capa/rules.py

```diff
@@ -256,10 +255,8 @@ def parse_feature(key: str):
     # keep this in sync with supported features
     if key == "api":
         return capa.features.insn.API
-    if key == "property/read":
-        return capa.features.insn.ReadProperty
-    if key == "property/write":
-        return capa.features.insn.WriteProperty
+    elif key == "property/read" or key == "property/write":
```

Comment:
move this to after `elif key == "namespace":` to keep historic order?

Reply:
Sure!